lua/dapui/windows/layout.lua

```diff
@@ -82,10 +84,23 @@ function WindowLayout:_area_size()
   return 0
 end
 
-function WindowLayout:resize()
+function WindowLayout:resize(width, height)
   if not self:is_open() then
     return
   end
+
+  -- Detecting whether self.area_state.size is a float or int
+  local num, fraction = math.modf(self.area_state.size)
+  if num == 0 and fraction > 0 then
+    if self.layout_type == 'vertical' then
```

Comment:
Add `---@field layout_type "horizontal" | "vertical"` to the `dapui.WindowLayout` type definition at the top

Reply:
Done